packages/@tests/smoke/web-cra-ts/test/index.ts

```diff
@@ -22,6 +23,12 @@ const test = async () => {
     await symlink(CDN_PUBLIC_PATH, join(publicPath, "source"));
   }
 
+  try {
```

Comment:
Can you please document what happens here and why?

Also, is it platform dependent in any way?

Reply:
I refactored it to self descriptive function.

Yes, it's platform independent